src/dom-to-image-more.js

```diff
@@ -1272,6 +1275,12 @@
         const targetStyle = targetElement.style;
 
         util.asArray(sourceComputedStyles).forEach(function (name) {
+            if (options.filterStyles) {
+                if (!options.filterStyles(name)) {
```

Comment:
Per @NogaMan's comment, we should be doing 
`if (!options.filterStyles(node, name)) {`